test/testffmpeg.c

```diff
@@ -137,18 +138,37 @@ static SDL_bool CreateWindowAndRenderer(Uint32 window_flags, const char *driver)
 
 #ifdef HAVE_EGL
     if (useEGL) {
-        const char *extensions = eglQueryString(eglGetCurrentDisplay(), EGL_EXTENSIONS);
-        if (SDL_strstr(extensions, "EGL_EXT_image_dma_buf_import") != NULL) {
-            has_EGL_EXT_image_dma_buf_import = SDL_TRUE;
+        const char *egl_extensions = eglQueryString(eglGetCurrentDisplay(), EGL_EXTENSIONS);
+        if (!egl_extensions) {
+            return SDL_FALSE;
+        }
+
+        char *extensions = SDL_strdup(egl_extensions);
+        if (!extensions) {
+            return SDL_FALSE;
         }
 
+        char *saveptr, *token;
+        token = SDL_strtok_r(extensions, " ", &saveptr);
```

Comment:
Is `token` guaranteed to be non-NULL here?

Reply:
I guess not, since `eglQueryString()` can return the empty string if there are no supported EGL extensions. Thanks!